src/Public/Caching.ps1

```diff
@@ -462,6 +462,7 @@ if (Test-PodeCacheStorage -Name 'ExampleStorage') { }
 function Test-PodeCacheStorage {
     [CmdletBinding()]
     param(
+        [Alias('Key')]
         [Parameter(Mandatory = $true)]
         [string]
         $Name
```

Comment:
`-Name` is the correct parameter here for `Test-PodeCacheStorage`. In the locations where `Test-PodeCacheStorage` is being erroneously called with `-Key` instead of `-Name`, those need to be updated to be `-Name`, rather than adding an Alias.